.github/workflows/Documenter.yaml

```diff
@@ -0,0 +1,34 @@
+name: Documentation
+
+on:
+  push:
+    branches:
+      - master
+    tags: '*'
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: 1.3
+      - name: Install MPI
+        run: |
+          sudo apt-get update
+          sudo apt-get install mpich libmpich-dev
+      - name: Install dependencies
+        run: julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
```

Comment:
```suggestion
        run: julia --project=docs/ -e 'using Pkg; Pkg.instantiate()'
```

Reply:
Since you commit the manifest you can pre-`dev` the `CLIMA` package as a relative path.

Reply:
ah, figured that out.